README.md

```diff
@@ -34,11 +38,11 @@ Here are all the supported values, including optional ones:
     {
       // Unique Open VSX extension ID in the form "<namespace>.<name>"
       "id": "rebornix.ruby",
-      // (OPTIONAL) The version to publish to Open VSX (defaults to the package.json version)
-      "version": "0.27.0",
       // Repository URL to clone and publish from
       "repository": "https://github.com/rubyide/vscode-ruby",
-      // (OPTIONAL) The Git branch, tag, or commit to check out before publishing (defaults to the repository's default branch)
+      // (RECOMMENDED) The version to publish to Open VSX (defaults to the package.json version)
```

Comment:
Calling it `RECOMMENDED` is a great idea, thanks! 👍 I also like the proposed new options order.

However, my original description was slightly wrong, so I might as well drive-by fix it. 😇

Explanation: This value is only used in comparison with what OpenVSX already has, to know if publishing should be skipped or not (but if publishing happens, it's the actual version string in `package.json` that will be used).

```suggestion
      // (RECOMMENDED) The version that should be published on Open VSX (defaults to the package.json version)
```